apps/dialtone-documentation/docs/components/icon.md

```diff
@@ -167,6 +167,8 @@ We encourage utilizing the [Stack component](/components/stack/) for aligning el
 
 ## Sizes
 
+Dialtone provides eight sizes for icons, three of them are **device only**. Each of the sizes represents the width and a height the icon is going to have:
```

Comment:
Not sure about the meaning of "device only", if this is outdated I can remove.

Reply:
I don't think this is a thing anymore.. you can remove.